tftypes/value.go

```diff
@@ -46,6 +46,10 @@ type Value struct {
 func (val Value) String() string {
 	typ := val.Type()
 
+	if typ == nil {
+		return "tftypes.Value<>"
```

Comment:
What if it we make it clearer that this is an _invalid_ Value, something that wasn't constructed as it was supposed to be?
```suggestion
		return "<invalid typeless tftypes.Value>"
```

Reply:
Good idea